test/unittests/commands/excel2json/test_list_node.py

```diff
@@ -0,0 +1,174 @@
+            nodes=[sub_1],
+        )
+        expected = {"name": "RootID", "labels": {"en": "root_label_en", "de": "root_label_de"}}
+        res = root._make_myself()
+        assert expected == res
+
+    def test_to_json(self) -> None:
+        sub_1 = ListNode(id_="Node_1", labels={"en": "Node_1_label_en"}, row_number=1)
+        sub_21 = ListNode(id_="SubNode_21", labels={"en": "SubNode_21_label_en"}, row_number=3)
+        sub_2 = ListNode(id_="Node_2", labels={"en": "Node_2_label_en"}, row_number=2, sub_nodes=[sub_21])
+        root = ListRoot(
+            id_="RootID",
+            labels={"en": "root_label_en", "de": "root_label_de"},
+            nodes=[sub_1, sub_2],
+        )
+        expected = {
+            "name": "RootID",
+            "labels": {"en": "root_label_en", "de": "root_label_de"},
+            "nodes": [
+                {"name": "Node_1", "labels": {"en": "Node_1_label_en"}},
+                {
+                    "name": "Node_2",
+                    "labels": {"en": "Node_2_label_en"},
+                    "nodes": [{"name": "SubNode_21", "labels": {"en": "SubNode_21_label_en"}}],
+                },
+            ],
+        }
+        res = root.to_json()
+        assert res == expected
+
+
+class TestListRootCreate:
+    def test_problem(self) -> None:
+        root = ListRoot.create(id_="", labels={}, comments={}, nodes=[])
+        assert isinstance(root, ListProblem)
+        assert root.root_id == ""
+        assert root.root_problems == {
+            "name": "The name of the list does not contain any characters.",
+            "labels": "At least one label per list is required.",
+            "list nodes": "At least one node per list is required.",
+        }
+
+    def test_wrong_language(self) -> None:
+        root = ListRoot.create(id_="str", labels={"ur": "label"}, comments={"ur": "comment"}, nodes=[])
+        assert isinstance(root, ListProblem)
+        assert root.root_id == "str"
+        assert root.root_problems == {
+            "labels": "Only the following languages are supported: 'en', 'de', 'fr', 'it', 'rm'.",
+            "comments": "Only the following languages are supported: 'en', 'de', 'fr', 'it', 'rm'.",
+            "list nodes": "At least one node per list is required.",
+        }
+
+    def test_id_na(self) -> None:
+        root = ListRoot.create(id_=pd.NA, labels={}, comments={}, nodes=[])  # type: ignore[arg-type]
+        assert isinstance(root, ListProblem)
+        assert root.root_problems == {
+            "name": "The name of the list may not be empty.",
+            "labels": "At least one label per list is required.",
+            "list nodes": "At least one node per list is required.",
+        }
+
+    def test_float(self) -> None:
+        nd = ListNode(id_="NodeID", labels={"en": "node_label_en"}, row_number=1)
+        root = ListRoot.create(id_=1.123, labels={"en": "node_label_en"}, comments={}, nodes=[nd])  # type: ignore[arg-type]
```

Comment:
It seems a bit odd to type `id_` as string, but then handling ints and floats, too. Either, you should allow ints and floats in the typing (because your code is able to handle them), or you should raise an error if they are passed in.